### PR TITLE
[API] Normalize headers in documentation

### PR DESCRIPTION
**Pull request by johnsaigle**
This pull request normalizes the API v3 documentation so that:

* Top-level headers are h1

* Second-level headers are h2

* Third level headers are h3

Also fixed small typos.

### CODE REVIEW

**Comment by victoriafoing** [comment]
I noticed that ### 2.1.3 Single project visits and ### 2.1.3 Single project candidates have the same header number. Could you edit this in this PR?

**Comment by johnsaigle** [comment]
@victoriafoing Good catch! Just fixed it :smile:

**Comment by victoriafoing** [comment]
It now skips 2.1.3! But once you change it to 2.1.3 and 2.1.4, this PR should be good to go.

**Comment by johnsaigle** [comment]
Oops! Just fixed it.

**Comment by driusan** [comment]
Could only change the target branch to minor through the web gui, needs to be rebased.

**Comment by zaliqarosli** [comment]
@driusan @johnsaigle i think the header levels make sense

File: docs/API/LorisRESTAPI_v0.0.3.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Loris API - v0.0.3-dev
 
-## 1.0 Overview
+# 1.0 Overview
```
[the remark this answers is not in the document]
Comment (johnsaigle) [reply docs/API/LorisRESTAPI_v0.0.3.md]: I'm just going to remove the title header instead

**Comment by johnsaigle** [comment]
@driusan The title is back.

**Comment by driusan** [comment]
@johnsaigle will merge pending travis passing..

**Comment by johnsaigle** [comment]
@driusan Can Travis fail documentation? Just curious :smile:

**Comment by driusan** [comment]
@johnsaigle Not at the moment, but green check marks make me feel safer

**Comment by johnsaigle** [comment]
:heavy_check_mark: :heavy_check_mark: :heavy_check_mark: :heavy_check_mark: :heavy_check_mark: :heavy_check_mark: :heavy_check_mark: :heavy_check_mark: :heavy_check_mark: :heavy_check_mark: :heavy_check_mark: :heavy_check_mark: :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

**Pull request merged**